pkgs/by-name/ho/hours/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
```

Comment:
Please use finalAttrs pattern.

Reply:
Isn't this an anti-pattern? https://github.com/NixOS/nixpkgs/issues/310373

Reply:
As far as I know, no.

Reply:
Okay then.